### DISCUSSION TITLE
Client configuration without configuration file

### DISCUSSION THREAD

**Discussion by maersk-** [radcli/radcli#101] [Q&A]
1. I'm using rc_add_config(), rc_read_dictionary(), rc_apply_config() instead of a configuration file. But I'm not able to configure the server part. I can only use rc_add_config(rh, "servers", "path/to/server/file", nullptr, 0). But I could not find out how to add the content of the servers file eg. "localhost testing123". Is there a way?
2. With rc_add_config() I can set the retries to 0 or not configure it at all and it works (just sends one request). If I do the same in the configuration file and use rc_read_config() I get an error. Why do the two methods not behave the same?

**Comment by maersk-** [comment]
1. I just checked out the code and found the following comment: auth/acct servers are added in the form: host[:port[:secret]]!

**Comment by nmav** [comment]
There are several examples in the src/ directory. The behavior should be the same whether you use add_config or setting a configuration file.
